Declarations/Contracts/TelemetryTypes/DependencyTelemetry.ts

```diff
@@ -34,7 +34,7 @@ export interface DependencyTelemetry extends Telemetry {
     /**
      * Result code returned form the remote component. This is domain specific and can be HTTP status code or SQL result code
      */
-    resultCode: string;
+    resultCode: string | number;
```

Comment:
Will we serialize it to string for JSON purposes, so that endpoint accepts the item?

Reply:
Yep! While converting to an Envelope it's converted to string.

I changed the type here because many pre-existing docs incorrectly assumed this to take a number. I figured it would be most beneficial to just actually fully support it as a number (before this it would silently fail as we don't expect number and would not convert to string).